third_party/libunwindstack/tests/PeCoffUnwindInfoUnwinderX86_64Test.cpp

```diff
@@ -333,13 +349,47 @@ class PeCoffUnwindInfoUnwinderX86_64Test : public ::testing::Test {
     new_unwind_info.unwind_codes = unwind_codes;
     new_unwind_info.exception_handler_address = 0L;
 
-    // TODO: Chained info.
-    new_unwind_info.chained_info = RuntimeFunction{0, 0, 0};
+    if (options.has_chained_info) {
+      new_unwind_info.chained_info = RuntimeFunction{0, 0, options.chained_info_offset};
+    }
 
     *unwind_info = new_unwind_info;
   }
 
+  uint64_t WriteUnwindInfoToFakeObjectFile(uint32_t file_offset, const UnwindInfo& unwind_info) {
```

Comment:
> FWIW if you preferred to mock the parsing I would suggest to create an interface for PeCoffUnwindInfos and provide a pointer to a MockPeCoffUnwindInfos to the unwinder instead of making the unwinder itself mockable. Probably irrelevant now, I just wanted to mention it.

Yeah, makes sense, I had considered that at some point, not sure why I opted for mocking the unwinder itself. In any case, thanks for the suggestions, I think it's in a better state now.